department-of-veterans-affairs/vets-website
# VAGOV-2225: fix "blue button" links

## Description
fixes the "blue button" links at the top of facility pages

## Testing done
locally with staging data

1. rebuild site
2. go to http://localhost:3001/pittsburgh-health-care/
3. click on the three blue buttons to make sure they go to:
- Make an appointment http://localhost:3001/pittsburgh-health-care/appointments
- View A-Z Services http://localhost:3001/pittsburgh-health-care/health-services
- Become a patient http://localhost:3001/pittsburgh-health-care/become-a-patient
4. go to a facility detail page and click on
- Pharmacy http://localhost:3001/pittsburgh-health-care/pharmacy (i don't think this page is created yet, but make sure the url is correct)

## Screenshots
n/a - urls

## Acceptance criteria
https://va-gov.atlassian.net/browse/VAGOV-2225

As a user on the VAMC-Pittsburgh site, I want access to scheduling, services, and enrollment content on the Facility and Contact Us pages so I can get to these patient task pages in one click/tap.

AC:

The following buttons, with correct page links, are at the top of the the following pages:

Follow the order of buttons listed below, from top to bottom, to display on the front-end, from left to right

Buttons should be placed under page intro text, except on the regional homepage (where they sit above the intro text)

Pittsburgh regional homepage: /pittsburgh-health-care

Make an appointment

View A-Z health services

Become a patient

Contact Us: /contact-us

Make an appointment

View A-Z health services

Become a patient

A-Z health services page: /health-services

Make an appointment

Become a patient

Pharmacy

University Drive facility page

Make an appointment

Become a patient

Pharmacy

H.J. Heinz facility page

Make an appointment

Become a patient

Pharmacy

Beaver facility page

Make an appointment

Become a patient

Pharmacy

Belmont facility page

Make an appointment

Become a patient

Pharmacy

Fayette facility page

Make an appointment

Become a patient

Pharmacy

Washington facility page

Make an appointment

Become a patient

Pharmacy

Westmoreland facility page

Make an appointment

Become a patient

Pharmacy

Button links:

Make an appointment: /pittsburgh-health-care/make-an-appointment

A-Z health services: /pittsburgh-health-care/health-services/

Become a patient: /pittsburgh-health-care/become-a-patient/

Pharmacy: /pittsburgh-health-care/pharmacy/

Implementation Notes:

These links are hard coded into the metalsmith templates as they will not change frequently ideal state is that within the front end templates, these are controlled from a central component, and don’t need to be updated on all pages.

## Definition of done
